src/helics/application_api/Subscriptions.hpp

```diff
@@ -20,33 +20,6 @@ SPDX-License-Identifier: BSD-3-Clause
 
 namespace helics {
 
-/** get a value directly from the subscription key name
-@details this is a convenience function to get a value directly from the subscription key name
-this function should not be used as the primary means of retrieving value as it does involve an
-additional map find operation vs the member getValue calls
-@param fed a reference to a valueFederate
-@param key  the name of the publication
-*/
-template<class X>
-X getValue(ValueFederate& fed, std::string_view key)
-{
-    return fed.getInput(key).getValue<X>();
-}
-
-/** get a value directly from the subscription key name
-@details this is a convenience function to get a value directly from the subscription key name
-this function should not be used as the primary means of retrieving value as it does involve an
-additional map find operation vs the member getValue calls
-@param fed a reference to a valueFederate
-@param key  the name of the publication
-@param obj the obj to store the retrieved value
-*/
-template<class X>
-void getValue(ValueFederate& fed, std::string_view key, X& obj)
-{
-    fed.getSubscription(key).getValue<X>(obj);
-}
-
```

Comment:
If these are part of the public API, we should update the changelog to note that they are gone now and what alternative function to use.

Reply:
The C++ API only,  so yes it was public but also not used as far as I can tell, and generated some deprecation warnings and I think they have nearly completely been superceeded by other operations.  And I agree the changelog will need to be updated.